app/models/user.rb

```diff
@@ -106,6 +106,32 @@ def gpo_verification_pending_profile?
     gpo_verification_pending_profile.present?
   end
 
+  def suspended?
+    suspended_at.to_s > reinstated_at.to_s
```

Comment:
why are we `.to_s`-ing these? Does that do something with `nil` values? My expectation is that we compare the object versions of these timestamps rather than the strings

Reply:
YYes! i added this because of nil values